app/models/searchs/vacancy.rb

```diff
@@ -3,45 +3,15 @@ def initialize(repository = ::Vacancy)
     @repository = repository
   end
 
-  def list(query = nil, lang = nil)
+  def list(query = nil)
     return @repository.recent if query.blank?
 
-    lang ||= I18n.locale == :'pt-BR' ? "portuguese" : "english"
+    query = query.split(' ').map { |entry| "#{entry}:*" }.join(' & ')
```

Comment:
nice to know, but I really thing is better explicit with space lol, i would need to google it everytime :P